py/specs/insert_bulk.toml

```diff
@@ -0,0 +1,13 @@
+[setup]
+statement_files = ["sql/crate_types.sql"]
+
+[[load_data]]
+target = "crate_types"
+source = "data/crate_types.json"
+
+bulk_size = 1000
+concurrency = 1
+num_records = 1000000
```

Comment:
you'd need to check in a `crate_types.json` file with 1M rows (which is about 180M
to be able to insert 1M rows with `load_data`

maybe use the existing `id_int_value_str.json`?